components/Input.js

```diff
@@ -2,20 +2,24 @@ import React from 'react';
 import { TextInput, View } from 'react-native';
 import PropTypes from 'prop-types';
 
-const Input = ({ text, onChange, name }) => (
+const Input = ({ value, onChange, name, placeholder }) => (
```

Comment:
Ah, that's where the Input component was coming from.

Reply:
Jess Jess Jess vury sneaky